db/schema.rb

```diff
@@ -997,8 +992,13 @@
     t.datetime "reset_password_sent_at"
     t.string   "api_key",                :limit => 40
     t.integer  "enterprise_limit",                     :default => 1, :null => false
+    t.string   "confirmation_token"
+    t.datetime "confirmed_at"
+    t.datetime "confirmation_sent_at"
+    t.string   "unconfirmed_email"
```

Comment:
@Matt-Yorkley If your dev db contained this migration at the time you ran rake `db:migrate`, I think these would have been added to your schema? I could be wrong but I seem to remember this happening to me before. The way schema.rb gets updated is still a bit of a black box to me...